> Task :clean
> Task :compileKotlin FROM-CACHE
> Task :compileJava FROM-CACHE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :compileTestKotlin FROM-CACHE
> Task :compileTestJava FROM-CACHE
> Task :processTestResources NO-SOURCE
> Task :testClasses UP-TO-DATE

> Task :test FAILED

RegistrationHandlerTests > returns_Created_with_body_on_success() FAILED
    java.lang.AssertionError at RegistrationHandlerTests.java:43

RegistrationHandlerTests > returns_Forbidden_for_blacklisted() FAILED
    java.lang.AssertionError at RegistrationHandlerTests.java:73

RegistrationHandlerTests > returns_Conflict_for_duplicate() FAILED
    java.lang.AssertionError at RegistrationHandlerTests.java:58

RegistrationHandlerTests > returns_InternalError_for_other_exceptions() FAILED
    java.lang.AssertionError at RegistrationHandlerTests.java:96

16 tests completed, 4 failed

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':test'.
> There were failing tests. See the report at: file:///Users/duncan/Documents/Projects/refactoring-to-kotlin-book/refactoring-to-kotlin-book/build/testing-history/build/reports/tests/test/index.html

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

